spec/requests/organization_requests_spec.rb

```diff
@@ -0,0 +1,171 @@
+require "rails_helper"
+
+RSpec.describe "Organizations", type: :request do
+  let(:default_params) do
+    { organization_id: @organization.to_param }
+  end
+
+  context "While signed in as a normal user" do
+    before do
+      sign_in(@user)
+    end
+
+    describe "GET #show" do
+      before { get organization_path(default_params) }
+
+      it { expect(response).to be_successful }
+
+      it 'load the current organization' do
+        expect(assigns(:organization)).to have_attributes(
+          name: @organization.name,
+          email: @organization.email,
+          url: @organization.url
+        )
+      end
+    end
+
+    describe "GET #edit" do
+      before { get edit_organization_path(default_params) }
+
+      it { expect(response).to have_http_status(:redirect) }
+      it { expect(response).to have_error }
+    end
+
+    describe "PATCH #update" do
+      let(:update_param) { { organization: { name: "Thunder Pants" } } }
+      before do
+        patch "/#{default_params[:organization_id]}/manage",
```

Comment:
There is another way to invoke the route?

Reply:
Hey there, it doesn't look like there is any route helper defined for this route. I think this is okay